### PR TITLE
Text to speach function

### PR DESCRIPTION
**Pull request by Rafalzie**

### CODE REVIEW

File: src/components/BackArrow.tsx
Comment (utlandingur) [inline src/components/BackArrow.tsx:1]: remove empty object/curly braces

File: src/components/options.tsx
Comment (utlandingur) [inline src/components/options.tsx:1]: remove unused hook and delete the hook

File: src/components/microphone.tsx
Comment (utlandingur) [inline src/components/microphone.tsx:1]: get rid of the needless handleonclick function. just pass directly.

File: src/hooks/useAudio.ts
Comment (utlandingur) [inline src/hooks/useAudio.ts:1]: delete this

File: src/providers/storyProvider.tsx
Comment (utlandingur) [inline src/providers/storyProvider.tsx:1]: fine but this should be in a separate branch

**Review by utlandingur — CHANGES_REQUESTED** [review]
make changes then ill approve

File: src/components/microphone.tsx
Comment (Rafalzie) [reply src/components/microphone.tsx:1]: done

File: src/components/options.tsx
Comment (Rafalzie) [reply src/components/options.tsx:1]: done i guess

File: src/hooks/useAudio.ts
Comment (Rafalzie) [reply src/hooks/useAudio.ts:1]: done

**Review by utlandingur — APPROVED** [review]

**Pull request merged**